src/test_l1_dsAudio.c

```diff
@@ -304,27 +405,27 @@ void test_l1_dsAudio_positive_dsGetAudioPort(void) {
 	UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
 
 	dsError_t  result;
-	intptr_t  handle[NUM_OF_PORTS] = {INT_ARRAY_INIT};
+	intptr_t  handle[numports];
```

Comment:
C supports variable-sized arrays from the C99 standard, will fail for previous standards. Better to use pointer and malloc.